### PR TITLE
Get Headlines Use case

### PR DESCRIPTION
**Pull request by peter-wachira**

### CODE REVIEW

**Review by zaloAustine — APPROVED** [review]
all good

**Comment by zaloAustine** [comment]
Check the errors

**Comment by peter-wachira** [comment]
> Check the errors

@zaloAustine thanks for the review mate.  You can proceed to merge, they are just linting checks by Codacy, they aren't a major consideration ATM.

File: app/src/main/java/com/droid/newsapiclient/InfoFragment.kt [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
+                              savedInstanceState: Bundle?): View? {
+        // Inflate the layout for this fragment
+        return inflater.inflate(R.layout.fragment_info, container, false)
+    }
+
+    companion object {
+        /**
+         * Use this factory method to create a new instance of
```
Comment (zaloAustine) [inline app/src/main/java/com/droid/newsapiclient/InfoFragment.kt:40]: clean up this redundant code

File: app/src/main/java/com/droid/newsapiclient/NewsFragment.kt [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        return inflater.inflate(R.layout.fragment_news, container, false)
+    }
+
+    companion object {
+        /**
+         * Use this factory method to create a new instance of
+         * this fragment using the provided parameters.
+         *
+         * @param param1 Parameter 1.
+         * @param param2 Parameter 2.
```
Comment (zaloAustine) [inline app/src/main/java/com/droid/newsapiclient/NewsFragment.kt:44]: same here

**Review by zaloAustine — COMMENTED** [review]
LGTM

**Pull request merged**